Swivl/Sniffs/Namespaces/UseDeclarationSniff.php

```diff
@@ -43,6 +43,18 @@ public function process(File $phpcsFile, $stackPtr): void
         $usePtr = $stackPtr;
         $lastUsePtr = null;
 
+        $nameTokens = [T_NS_SEPARATOR, T_STRING];
+
+        if (defined('T_NAME_QUALIFIED')) {
+            $nameTokens[] = T_NAME_QUALIFIED;
+        }
+        if (defined('T_NAME_FULLY_QUALIFIED')) {
+            $nameTokens[] = T_NAME_FULLY_QUALIFIED;
+        }
+        if (defined('T_NAME_RELATIVE')) {
+            $nameTokens[] = T_NAME_RELATIVE;
+        }
```

Comment:
It seems we can simplify that and make it more maintenable:
```suggestion
        $nameTokens = Tokens::NAME_TOKENS + [T_NS_SEPARATOR => T_NS_SEPARATOR];
```